ytmusicapi/parsers/playlists.py

```diff
@@ -94,8 +94,11 @@ def parse_playlist_items(results, menu_entries: Optional[List[List]] = None, is_
         }
 
         if is_album:
-            track_idx_found = nav(data, ["index", "runs", 0, "text"], True)
-            song["track_number"] = track_idx_found if track_idx_found is None else int(track_idx_found)
+            if not isAvailable:
+                song["track_number"] = None
+            else:
+                # returning None from nav is pointless as the keys are always present
+                song["track_number"] = int(nav(data, ["index", "runs", 0, "text"]))
```

Comment:
```suggestion
            song["track_number"] = int(nav(data, ["index", "runs", 0, "text"])) if isAvailable else None
```

a bit more readable imo

Reply:
i wouldn't touch nav because it's got like 200 usages across the project and side effects are not unlikely